src/clib/xtg/surf_export_zmap_ascii.c

```diff
@@ -60,7 +60,7 @@ surf_export_zmap_ascii(FILE *fc,
 
     /* local declarations */
     int i, j, nn, fcode;
-    float myfloat, xmax, ymax;
+    double myfloat, xmax, ymax;
```

Comment:
Name it `mydouble` or `dblvalue`?

Reply:
Both should be fine, `fprintf` of float is converted to double before it is printed (but could be loss of information in that conversion). For `fscanf` it would have to be `lf`.